src/daemon/rpmostree-sysroot-core.c

```diff
@@ -385,6 +385,59 @@ rpmostree_syscore_add_deployment (OstreeSysroot      *sysroot,
   return g_steal_pointer (&new_deployments);
```

Comment:
The commit message should be `s/rpmostree_syscore_filter_deployments/rpmostree_syscore_add_deployment/` right?